### ISSUE TITLE
Incorrect cookie name results in silent failure

### ISSUE THREAD

**Issue by dhirschfeld** [rocker-org/rocker-versioned2#165]
https://github.com/rocker-org/rocker-versioned2/blob/4d40a1c4a01e62f9bed0294cc6229facb39e6377/scripts/install_rstudio.sh#L70-L71

After installing `rstudio-server-1.4.1106-amd64.deb` the cookie filename I see is `secure-cookie-key` instead of `secret-cookie-key`
```bash
❯ ls /var/lib/rstudio-server/
drwxr-xr-x   - root           31 May 10:49 body
drwxr-xr-x   - root           31 May 10:49 conf
drwxr-xr-x   - root           31 May 10:49 proxy
.rw-r--r-- 20k rstudio-server 31 May 10:49 rstudio.sqlite
.rw-------  32 root           31 May 10:49 secure-cookie-key
```

**Comment by eitsupi** [comment]
This problem was reported in #137, at which point the file name was written as `secure-cookie-key`.
In other words, the file name has not been changed, but the modification made in 415b065d45666bbcfb435268bf30b0bfc4f10c48 seems to have specified the wrong file name.

**Comment by dhirschfeld** [comment]
Right, I think the TL;DR is that the change in 415b065d45666bbcfb435268bf30b0bfc4f10c48 is effectively a no-op and as such users are still exposed to the issue identified in #137.

I've updated the issue name to be more descriptive./accurate

**Comment by cboettig** [comment]
@dhirschfeld right on, thanks for catching.  will fix this

**Comment by cboettig** [comment]
(also still unclear to me why RStudio is creating this file on install, as you can see we do not run RStudio executable in the install so it should not be created in the first place.)

**Comment by dhirschfeld** [comment]
> *also still unclear to me why RStudio is creating this file on install, as you can see we do not run RStudio executable in the install so it should not be created in the first place.*

Yeah, I think this *should* be fixed upstream - it seems to have no benefit and admits the potential of a hard to identify security issue :/

**Comment by dhirschfeld** [comment]
Fixed by #166

**Comment by cboettig** [comment]
Re upstream, @kevinushey just pointed out in https://github.com/rstudio/rstudio/issues/9175#issuecomment-852388099 that RStudio does indeed seem to be executed by the debian postinstall script, just as @eddelbuettel first suggested might be the case back in #137.   Anyway, #166 should take care of this for us (once images are rebuilt) and :crossed_fingers: it is also fixed upstream
